src/handlebars/icedtea-web.handlebars

```diff
@@ -0,0 +1,57 @@
+{{> header title='IcedTea-Web | ' }}
+
+<!-- TODO Style should match installation page, it doesn't quite... -->
+<main class="grey-bg upstream">
+
+  <h1 class="margin-bottom large-title">IcedTea-Web Builds</h1>
+
+  <div class="info-page-container upstream blue-link-section">
+
+    <div class="callout">
+      <h3 class="bold">What are these binaries?</h3>
+      <p>The IcedTea-Web Javaws_splash project provides a Free Software web browser plugin running applets written in the Java™ programming language and an implementation of Java™ Web Start, originally based on the NetX project.</p>
```

Comment:
I'm not sure I like this wording. The main thing we want to push is the web start stuff (javaws command line) so talking about this purely in terms of the plug (which is hard to use with any modern browser now!) is probably not the best headline to use.

Reply:
Appel support will be removed soon ( in ITW 2.0 ) So it is better to not speak about.
https://github.com/AdoptOpenJDK/IcedTea-Web/pull/162
And AdoptInstaller dont register anything about applet in browser.

Reply:
removed